src/main/java/com/evacipated/cardcrawl/mod/stslib/actions/common/SelectCardsAction.java

```diff
@@ -13,91 +14,89 @@
-    * @param anyNumber - whether player has to select exact number of cards (amount) or any number up to, including 0.
-    * false for exact number.
-    * @param cardFilter - Filters the cards in the group.
-    * Example: if you want to display only skills, it would be c -> c.type == CardType.SKILL
-    * If you don't need the filter, set it as c -> true
-    * @param callback - What to do with cards selected. Accepts a list with cards selected. The list would only contain one element if it's "Select one card"
-    * Example: if you want to lose 1 hp and upgrade each card selected, it would look like
-    * list -> {
-    * addToBot(
-    * new LoseHPAction(player, player, list.size());
-    * list.forEach(c -> c.upgrade());
-    * )}
-    *
-    * if there's no callback the action will not trigger simply because you told player to "select cards to do nothing with them"
-    * */
+     * @param group         - ArrayList of cards to filter and select from.
+     *                      Example: AbstractDungeon.player.discardPile.group
+     * @param amount        - maximum number of cards allowed for selectoin
+     * @param textForSelect - text that will be displayed on the grid select screen at the bottom. It will show just this text with nothing else added by itself.
+     * @param anyNumber     - whether player has to select exact number of cards (amount) or any number up to, including 0.
+     *                      false for exact number.
+     * @param centered      - whether or not to center the GridSelectScreen's cards.
+     * @param cardFilter    - Filters the cards in the group.
+     *                      Example: if you want to display only skills, it would be c -> c.type == CardType.SKILL
+     *                      If you don't need the filter, set it as c -> true
+     * @param callback      - What to do with cards selected. Accepts a list with cards selected. The list would only contain one element if it's "Select one card"
+     *                      Example: if you want to lose 1 hp and upgrade each card selected, it would look like
+     *                      list -> {
+     *                      addToBot(
+     *                      new LoseHPAction(player, player, list.size());
+     *                      list.forEach(c -> c.upgrade());
+     *                      )}
+     *                      <p>
+     *                      if there's no callback the action will not trigger simply because you told player to "select cards to do nothing with them"
+     *
+     */
 
-    public SelectCardsAction(ArrayList<AbstractCard> group, int amount, String textForSelect, boolean anyNumber, Predicate<AbstractCard> cardFilter, Consumer<List<AbstractCard>> callback)
-    {
+    public SelectCardsAction(ArrayList<AbstractCard> group, int amount, String textForSelect, boolean anyNumber, boolean centered, Predicate<AbstractCard> cardFilter, Consumer<List<AbstractCard>> callback) {
```

Comment:
This change will break any mods using `SelectCardsAction(ArrayList<AbstractCard> group, int amount, String textForSelect, boolean anyNumber, Predicate<AbstractCard> cardFilter, Consumer<List<AbstractCard>> callback)` overload, I think you should re-add the overload without `boolean centered`.